src/PSDesiredStateConfiguration/PSDesiredStateConfiguration.psm1

```diff
@@ -4627,7 +4627,7 @@ function Invoke-DscClassBasedResource
     Write-Debug "Importing $path ..."
     $powershell = [PowerShell]::Create('CurrentRunspace')
     $script = @"
-using module $path
+using module '$path'
```

Comment:
```suggestion
using module "$path"
```

Reply:
It's a here-string, this does not make a difference